cmd/kubeadm/app/cmd/options/constant.go

```diff
@@ -84,3 +84,24 @@ const CSROnly = "csr-only"
 
 // CSRDir flag sets the location for CSRs and flags to be output
 const CSRDir = "csr-dir"
+
+// Token flags sets both the discovery-token and the tls-bootstrap-token when those values are not provided
+const Token = "token"
```

Comment:
i think someone submitted a PR for this earlier and it merged.
possible cause for the merge conflict.

Reply:
+1